mate-panel/applet.h

```diff
@@ -111,11 +111,6 @@ void        mate_panel_applet_menu_set_recurse (GtkMenu     *menu,
 
 gboolean    mate_panel_applet_toggle_locked  (AppletInfo *info);
 
-void mate_panel_applet_position_menu (GtkMenu   *menu,
```

Comment:
Is not this API break? Should not *.so version of library be updated to reflect this?

Reply:
What library is this part of? I thought it was just part of the `mate-panel` application.

Reply:
Oh, sorry... I was responding based on function name / prefix that is used by `libmate-panel-applet`.

Reply:
Ah, yes, that is confusing. Clearing that up is another benefit of this PR.